SwiftyMock/Classes/ReactiveCocoa/ReactiveExtensions.swift

```diff
@@ -6,53 +6,40 @@
 // MARK: Reactive Function Call Mock/Stub/Spy
 
 import ReactiveSwift
+import Result
 
-open class ReactiveCall<Arg, Value, Err: Error>: FunctionCall<Arg, Value> {
-    open fileprivate(set) var stubbedError: Err?
-    open func fails(_ error: Err) {
-        stubbedError = error
-    }
-    
-    public override init() {
-        super.init()
-    }
-}
+// MARK: - Reactive Call Mock/Stub/Spy
+
+public typealias ReactiveCall<Arg, Value, Err: Error> = FunctionCall<Arg, Result<Value, Err>>
+public typealias ReactiveVoidCall<Value, Err: Error> = FunctionCall<Void, Result<Value, Err>>
```

Comment:
shouldn't we use `FunctionVoidCall` for typealiasing `ReactiveVoidCall` instead of `FunctionCall`?